cmd/notary-server/main.go

```diff
@@ -32,6 +23,10 @@ import (
 const (
 	jsonLogFormat = "json"
 	DebugAddress  = "localhost:8080"
+	// This is the generally recommended maximum age for Cache-Control headers
+	// (one year, in seconds, since one year is forever in terms of internet
+	// content)
+	maxMaxAge = 60 * 60 * 24 * 365
```

Comment:
I can definitely move it - I wasn't sure if it should go there, since it's only used for the server config, and it's an int of seconds instead of a duration (since the header is in ints, and the config should probably be an int since that's what people are probably used to if they want to set a cache control header)